Description: Move D-Bus policy file to /usr/share/dbus-1/system.d/
Origin: other, https://gitlab.gnome.org/GNOME/NetworkManager-pptp/merge_requests/6
Bug: https://gitlab.gnome.org/GNOME/NetworkManager-pptp/issues/1
Reviewed-By: Douglas Kosovic <doug@uq.edu.au>
Last-Update: 2019-03-18

This is supported since dbus 1.9.18.
The old location in /etc/dbus-1/system.d/ has been deprecated.

https://lists.freedesktop.org/archives/dbus/2015-July/016746.html
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index f3df202..dc7179e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -20,7 +20,7 @@ SUBDIRS = \
 plugindir = $(libdir)/NetworkManager
 plugin_LTLIBRARIES =
 
-dbusservicedir = $(sysconfdir)/dbus-1/system.d
+dbusservicedir = $(datadir)/dbus-1/system.d
 dbusservice_DATA = nm-l2tp-service.conf
 
 nmvpnservicedir = $(NM_VPN_SERVICE_DIR)
